src/relay/limiter/fixed.rs

```diff
@@ -5,19 +5,22 @@ use lru::LruCache;
 use std::{sync::Arc, time::UNIX_EPOCH};
 use tokio::sync::Mutex;
 
+/// FixedWindowOptions are used to determine user limits for each time window, and the size of that window.
 pub struct FixedWindowOptions {
     pub size: usize,
     pub count: usize,
     pub window: usize, // window in seconds
 }
 
+/// Counters are used to determine the consumption of some user at some time window started at `start`.
```

Comment:
a user during a time window started at `start` and please clarify if that start a timestamp?